modules/table/src/integrationTest/java/org/apache/ignite/internal/rebalance/ItRebalanceDistributedTest.java

```diff
@@ -500,7 +500,7 @@ void testDestroyPartitionStoragesOnEvictNode() throws Exception {
     @Test
     @UseTestTxStateStorage
     @UseRocksMetaStorage
-    @Disabled("https://issues.apache.org/jira/browse/IGNITE-20210")
+    @Disabled("https://issues.apache.org/jira/browse/IGNITE-19170")
     void testDestroyPartitionStoragesOnRestartEvictedNode(TestInfo testInfo) throws Exception {
```

Comment:
Should we write some tests that will ensure that proposed implementation is correct?

Reply:
On this step I mostly care not to break something, but probably there should be some tests for storages cleanup. I expect more tests to be added in [IGNITE-20993](https://issues.apache.org/jira/browse/IGNITE-20993)